discovery_engine_flutter/example/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
Where you using flutter 2.5? I realized I was using flutter 2.10

Reply:
me too, I'll downgrade and check again

Reply:
the bumped versions work for 2.5.3 and for 2.10, the old ones work only for 2.5.3, so let's maybe keep the bumped ones